python/README.md

```diff
@@ -0,0 +1,88 @@
+# Remote BMI
+
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
+[![Codecov test coverage](https://codecov.io/gh/eWaterCycle/remotebmi/graph/badge.svg)](https://app.codecov.io/gh/eWaterCycle/remotebmi)
+
+The [Basic Model Interface (BMI)](https://bmi.readthedocs.io/en/stable/) is a standard interface for models. 
+The interface is available in different languages and a [language agnosting version in SIDL](https://github.com/csdms/bmi/blob/stable/bmi.sidl).
+
+To have a consumer of the model and the provider of the model seperated you can use [grpc4bmi](https://grpc4bmi.readthedocs.io/), but this only works on languages that have a grpc implementation.
+This Python package replaced the gRPC protocol with an REST API.
+The [REST API specification](https://github.com/eWaterCycle/remotebmi/blob/main/openapi.yaml) is in the [OpenAPI](https://swagger.io/specification/) format.
+
+Remotebmi is available in other languages see [here](https://github.com/eWaterCycle/remotebmi?tab=readme-ov-file#structure).
+
+## Difference from BMI
+
+- Request body and response body are in JSON format
+- On errors you get 4xx and 5xx responses with [Problem Details](https://tools.ietf.org/html/rfc7807) as response body
+- Variable names must be URL safe
```

Comment:
I hope Julia people comply with this. Although it should go OK if they follow [the standard names](https://csdms.colorado.edu/wiki/CSDMS_Standard_Names).

Reply:
Although this is the python readme, so that should be fine. Not sure if it's possible to write a variable name in python that isn't url safe.